src/components/atoms/VolumeControl/VolumeControl.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback } from "react";
+import { faVolumeMute, faVolumeUp } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+
+import {
+  DEFAULT_AUDIO_VOLUME,
+  MAX_AUDIO_VOLUME,
+  MIN_AUDIO_VOLUME,
+} from "settings";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./VolumeControl.scss";
+
+export interface VolumeControlProps {
+  className?: string;
+  label?: string;
+  name: string;
+  muted?: boolean;
+  onMute?: () => void;
+  onVolume?: (volume: number) => void;
+  volume?: number;
+  withMute?: boolean;
+  withSlider?: boolean;
+}
+
+export const VolumeControl: React.FC<VolumeControlProps> = ({
+  className = "",
+  label = "",
+  name,
+  muted,
+  onMute,
+  onVolume,
+  volume = DEFAULT_AUDIO_VOLUME,
+  withMute = false,
+  withSlider = false,
+}) => {
+  const onSlider = useCallback(
+    (event) => {
+      const volume = Number(event.target.value);
+      return onVolume?.(volume);
+    },
+    [onVolume]
+  );
+
+  const parentClasses = classNames({
+    VolumeControl: true,
+    VolumeControl__container: true,
+    [className]: className,
+  });
```

Comment:
can we do something like this:
```suggestion
  const parentClasses = classNames("VolumeControl", { className });
```

why do we need 2 class names for the same div?

Reply:
I've shared this before: https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1628775557030900 with the first reference being https://github.com/sparkletown/sparkle/pull/1705#discussion_r687619209

TL;DR:
One `className` prop, multiple classes per element, a single id per document element, like HTML+CSS is expected to have.

Longer form:
Two classes (not class names) are needed for the same reason Bootstrap uses both `btn` and `btn-primary` at the same element and for the same reason there is B in BEM :
 
a reference to the top level element that will make it easier to (re-)style elements inside components from the outside without having to research/update each time what their internal specifics are or if they have changed meanwhile.

Reply:
Brand new className, now with minues one E class and plus 3 M ones @ eda678a6b658cc936def13246a4656c0b906b30d